terragrunt/aws/cloudfront/s3.tf

```diff
@@ -13,7 +13,9 @@ data "aws_iam_policy_document" "website_bucket_policy_doc" {
       identifiers = ["${aws_cloudfront_origin_access_identity.origin_access_identity.iam_arn}"]
     }
     actions = [
-      "s3:GetObject"
+      "s3:GetObject",
+      "s3:ListBucket"
```

Comment:
Unfortunately this won't work because the `s3:ListBucket` action does not allow a `resources` block that includes object ARNs (the trailing `/*`).

You'll need to create a new `statement` block in the policy like so:
```hcl
  statement {
    effect = "Allow"
    principals {
      type        = "AWS"
      identifiers = ["${aws_cloudfront_origin_access_identity.origin_access_identity.iam_arn}"]
    }
    actions = [
      "s3:ListBucket"
    ]
    resources = [
      "${each.value.s3_bucket_arn}"
    ]
  }
```